meteor/server/api/translationsBundles.ts

```diff
@@ -0,0 +1,46 @@
+import { TranslationsBundles, TranslationsBundleId } from '../../lib/collections/TranslationsBundles'
+import { TranslationsBundle, TranslationsBundleType } from 'tv-automation-sofie-blueprints-integration'
+import { getRandomId, unprotectString } from '../../lib/lib'
+import { logger } from '../logging'
+import { BlueprintId } from '../../lib/collections/Blueprints'
+
+export function upsertBundles(bundles: TranslationsBundle[], parentBlueprintId: BlueprintId) {
+	for (const bundle of bundles) {
+		const { type, language, data } = bundle
+
+		if (type !== TranslationsBundleType.I18NEXT) {
+			throw new Error(`Unknown bundle type ${type}`)
+		}
+
+		const namespace = unprotectString(parentBlueprintId)
```

Comment:
So I'm confused about the "namespace". If it is equal to the _id of the blueprint that created it, I think it should be named `blueprintId`